libs/s25main/network/GameServer.cpp

```diff
@@ -430,13 +434,112 @@ void GameServer::Stop()
     LOG.write("server state changed to stop\n");
 }
 
+// Check if there are players that have not been assigned a team but only a
+// range. Those players are assigned a team now while we try to balanace the
+// number of players per team. Returns true iff players have been assigned.
+static bool adjustSmartTeams(std::vector<JoinPlayerInfo>& playerInfos)
+{
+    std::set<unsigned> nPotentialPlayers[4];
```

Comment:
Could this get a better name? It isn't the number of players but player indices so remove the "n" prefix?

Reply:
sure.